content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -106,7 +109,45 @@ The max number of pods that is created within a single polling period. If there
 * **Target Average Value:** The number of messages that will be consumed on a job. It is defined on the scaler side. e.g. `queueLength` on `Azure Storage Queue` scaler.
 * **Running Job Count:** How many jobs are running.
 * **Number of the Scale:** The number of the job that is created.
+```yaml
```

Comment:
Please put empty line and a line before this section, eg:

```yaml

---

scalingStrategy: "default"                # Optional. Default: default. Which Scaling Strategy to use. 
```

Reply:
Done